<?xml version="1.0"?>
<launch>
    <arg name="current_map_name" default="map"/>
    <param name="map_name"    value="$(arg current_map_name)"/>

    <include file="$(find anscer_multimap_navigator)/launch/include/common.launch.xml"/>

    <!-- dynamic map changer node -->
    <node pkg="anscer_multimap_navigator" type="dynamic_map_changer" name="dynamic_map_changer"/>

    <!-- navigator action server node -->
    <node pkg="anscer_multimap_navigator" type="navigator_action_server" name="navigator_action_server"/>

    <!-- rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find anscer_navigation)/rviz/anscer_navigation.rviz"/>
    
    <include file="$(find anscer_navigation)/launch/navigation/dwa_navigation.launch"/>
    <include file="$(find anscer_navigation)/launch/localization/amcl_localization.launch"/>
</launch>
